src/lib/seed-db.js

```diff
@@ -15,18 +16,31 @@ function * setupHoldAccount () {
   }
 }
 
+function * hashPassword (password) {
+  return yield new Promise((resolve, reject) => {
+    bcrypt.genSalt(10, (error, salt) => {
```

Comment:
I had to lookup the meaning of `10` here and it's the number of rounds (defaults to 10). This is a case where I'd argue for an explaining variable (`const rounds = 10`) to pass into the function. But LGTM otherwise